.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,11 @@
-Description:
-<!--
-Replace this template with your PR description.
-Please remember to keep in mind the security levels outlined in
-[CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md) and to
-include a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title
-
-* **no**: None
-* **low**: Low chance of potential impact to, or exposure of patient data
-* **moderate**: Moderate chance of potential impact to, or exposure of patient data
-* **severe**: Severe chance of potential impact to, or exposure of patient data
-
-Please also:
-
-* Get thumbs from reviewer(s)
-* Verify all tests go green, including CI tests
--->
-
+**Description**
 
 ---
 **PR checklist**
 
-- [ ] This PR meets the Acceptance Criteria in the JIRA story
-- [ ] The JIRA story has been moved to Dev Review
-- [ ] This PR includes appropriate unit tests
-- [ ] I have added explanatory comments where the logic is not obvious
-- [ ] I have run and tested this change locally, and my testing process is described here
+- [ ] If this PR is associated with a Jira issue, I have included an issue number tag in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md)
+- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md)
+- [ ] I have manually tested this change
+- [ ] This PR includes appropriate automated tests, and I have documented any behavior that cannot be tested with code
 - [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
```

Comment:
Note: It would be nice if, someday, this were an automated PR check for feature flag modification that reminded us to do this.